docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Limit Trie Logs for Bonsai
+sidebar_position: 12
+description: Enable this early access feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Limit Trie Logs for Bonsai
+## Step by Step Guide
+
+:::caution
+
+We strongly recommend reading the rest of this documentation before running these commands because your node’s configuration may require changes to these commands.
+
+:::
+
+*Targeting 24.1.2 release, but currently you must be on the Besu `main` branch for this to work*
```

Comment:
I think we should wait on this page until the feature is released.

Reply:
removed this line